### PR TITLE
Mainnet deploy params and migration to enable market

### PR DESCRIPTION
**Pull request by jflatow**
We deploy with supply caps and rewards speeds set to 0, then create a proposal to:

 - Raise supply caps
 - Set rewards speed
 - Stream COMP to the rewards contract
 - Seed USDC reserves from Timelock
 
 This extracts just the deploy/proposal from #510, which can now be closed.

### CODE REVIEW

File: deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      configurator,
+      cometAdmin,
+      rewards,
+      USDC,
+      COMP,
+      WBTC,
+      WETH,
+      UNI,
+      LINK
+    } = await deploymentManager.getContracts();
```
Comment (jflatow) [inline deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts:26]: @scott-silver this raison d'etre for `getContracts`

**Review by scott-silver — APPROVED** [review]
🍾 🥳 🎉

**Comment by jflatow** [comment]
As an update, we are now just waiting for an update on recommended parameters from Gauntlet before making the deploy

**Comment by jflatow** [comment]
> As an update, we are now just waiting for an update on recommended parameters from Gauntlet before making the deploy

The commit now includes the updated parameters recommendations from Gauntlet

File: deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    const actions = [
```
Comment (kevincheng96) [inline deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts:29]: Nice, looks a lot cleaner like this 👍

File: deployments/mainnet/usdc/deploy.ts [hunk 3]
```diff
@@ -11,14 +11,16 @@ export default async function deploy(deploymentManager: DeploymentManager, deplo
...
-    [await comet.governor(), comet.address, WETH.address]
-  );
+  // const bulker = await deploymentManager.deploy(
+  //   'bulker',
+  //   'Bulker.sol',
+  //   [await comet.governor(), comet.address, WETH.address]
+  // );
 
-  return { ...deployed, bulker };
+  // return { ...deployed, bulker };
```
Comment (kevincheng96) [inline deployments/mainnet/usdc/deploy.ts:23]: What's the plan for deploying bulker? Probably in a new branch/PR, we could comment out everything other than deploying bulker and do the deploy from there?

File: deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, WBTC.address, exp(2_000, 8)],
+      }, {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, WETH.address, exp(25_000, 18)],
+      }, {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, UNI.address, exp(250_000, 18)],
```
Comment (kevincheng96) [inline deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts:46]: This one looks off. UNI is ~$8.5 right now (same price as LINK basically), so supply cap should be around 1mn as well.

File: deployments/mainnet/usdc/deploy.ts (hunk 3 above)
Comment (jflatow) [reply deployments/mainnet/usdc/deploy.ts:23]: We shouldn't need to comment the other lines

Comment (kevincheng96) [reply deployments/mainnet/usdc/deploy.ts:23]: Oh right, that's the magic of idempotent deploys 🪄

File: deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts (hunk 4 above)
Comment (jflatow) [reply deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts]: Updated to 1M UNI

**Review by kevincheng96 — APPROVED** [review]
Looks good 👍

**Comment by kevincheng96** [comment]
Now that this proposal has been executed, what's the next steps? Delete the migration script and commit the roots?

**Comment by jflatow** [comment]
> Now that this proposal has been executed, what's the next steps? Delete the migration script and merge the rest?

Going to merge this, then we delete and merge a new PR, I think. We said all migrations should make it to `main` if they actually occur

**Pull request merged**